servers.json

```diff
@@ -67,9 +67,9 @@
     },
     {
         "id": "neptuniummc",
-        "name": "NeptuniumMC Italia",
+        "name": "NeptuniumMC | Server Italiano",
```

Comment:
Hey, this name seems a bit too long, could you possibly shorten this down to `NeptuniumMC Italiano`?